### DISCUSSION TITLE
Parse `Bytes` from `RawForm`

### DISCUSSION THREAD

**Discussion by maxwell8888** [hecrj/iced#1805] [Q&A]
When using `RawForm` like so from the docs:
```rust
async fn handler(RawForm(form): RawForm) {}
let app = Router::new().route("/", get(handler));
```
`form` is a `Bytes`. I want to parse the `Bytes` into a usable form like a vector of tuples or HashMap. Before I add another library to the project for this, I was wondering is there is any functionality within Axum to parse the `Bytes`?

**Comment by hecrj** [comment]
Wrong library?

**Reply by maxwell8888** [reply]
haha yes, sorry!
